src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/GVMDependenciesNode.cs

```diff
@@ -186,23 +191,10 @@ public override IEnumerable<CombinedDependencyListEntry> SearchDynamicDependenci
                             continue;
                     }
 
-                    overrideTypeCur = potentialOverrideType;
```

Comment:
Couldn't figure out when this loop would have a meaningful impact. We could consider not checking in this part of the change and keep the loop, but I don't like to have code around that nobody understands and we have no test coverage for (there's apparently no failing test with this deleted even though we test generic virtual methods in the Generics test and in the DynamicGenerics test).